apps/meteor/client/components/Sidebar/Sidebar.tsx

```diff
@@ -3,7 +3,18 @@ import type { FC } from 'react';
 import React from 'react';
 
 const Sidebar: FC = ({ children, ...props }) => (
-	<Box display='flex' flexDirection='column' h='full' justifyContent='stretch' {...props} role='navigation'>
+	<Box
+		backgroundColor='surface-light'
+		borderColor='stroke-extra-light'
+		borderInlineEndWidth='1px'
```

Comment:
where have a token called default to use instead 1px

Reply:
Fixed